### DISCUSSION TITLE
What is the "Gradle Versioning Scheme"?

### DISCUSSION THREAD

**Discussion by kolloch** [renovatebot/renovate#23938] [Request Help]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

v15.28.2

### If you're self-hosting Renovate, select which platform you are using.

gitlab.com

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

Update the artifacts that are referenced in the `WORKSPACE` file via `maven_install` in the `artifacts` argument:

```starlark
maven_install(
    artifacts = [
        ## START: AUTOMANAGED DEPENDENCIES
        # ...
        "com.amazonaws:aws-java-sdk-core:1.12.515",
        "com.amazonaws:aws-java-sdk-dynamodb:1.12.515",
        "com.amazonaws:aws-java-sdk-kms:1.12.515",
        "com.amazonaws:aws-java-sdk-s3:1.12.515",
        "com.amazonaws:aws-java-sdk-sts:1.12.515",
        # ...
        ## END: AUTOMANAGED DEPENDENCIES
)
```

Note that I included the `## START: AUTOMANAGED DEPENDENCIES` comments here which are used by a migration script in case they make a difference. Of course there are more dependencies than shown.

### What you tried so far.

I am using https://bazel.build/ `6.3.2` with `maven_install` from https://github.com/bazelbuild/rules_jvm_external/ version 5.3.

There are still old gradle build files in the repository -- maybe that confuses renovate. It seems to detect outdated versions in those but not the `maven_install` artifacts versions.

### Relevant debug logs

There are some errors about updating the gradle dependencies -- I would like to just disable gradle integration but haven't found out how.

**Comment by rarkins** [comment]
The title of your post and the last sentence seem detached from the rest of your post. Please start with that problem or rewrite your post. In particular elaborate on "there are some errors"
